src/phrases/phrases-en.ts

```diff
@@ -179,6 +179,23 @@ const phrases = {
       onHoldFilter: "Do not count on hold",
       loading: "Loading...",
     },
+    subjectView: {
+      lma: "Subject view LMA-subjects",
+      da: "Subject view DA-subjects",
+      sa: "Subject view SA-subjects",
+      favorites: "Subject view favorite subjects",
+      description: "Overview of statistics related to resources in your subjects",
+      heart: "Hearts",
+      heartDescription: "Count of favorited resources",
+      flow: "In flow",
+      flowDescription: "Count of resources in flow",
+      old: "Outdated",
+      oldDescription: "Count of resources that have not been revised in over 5 years",
+      revision: "Revisions",
+      revisionDescription: "Count of resourcecs with expired revision date",
```

Comment:
```suggestion
      revisionDescription: "Count of resources with expired revision date",
```

Reply:
Synes "Number of resources" klinger bedre i mine ører.

Reply:
Enig! trodde jeg oppdaterte til det 😅